From 306059dcc636fc81aa72f1909852634f6a46be40 Mon Sep 17 00:00:00 2001
From: Jianjun Zhu <jianjun.zhu@intel.com>
Date: Fri, 1 Dec 2017 13:16:24 +0800
Subject: [PATCH 1/2] Use OpenSSL for usrsctp.

---
 usrsctp/BUILD.gn | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/usrsctp/BUILD.gn b/usrsctp/BUILD.gn
index e505d33..583c8ff 100644
--- a/usrsctp/BUILD.gn
+++ b/usrsctp/BUILD.gn
@@ -3,6 +3,7 @@
 # found in the LICENSE file.
 
 import("//build/toolchain/toolchain.gni")
+import("//build_overrides/ssl/ssl.gni")
 
 config("usrsctp_config") {
   include_dirs = [
@@ -130,7 +131,11 @@ static_library("usrsctp") {
     defines += [ "NON_WINDOWS_DEFINE" ]
   }
 
-  deps = [
-    "//third_party/boringssl",
-  ]
+  if (woogeen_use_openssl) {
+    configs += [ "//build_overrides/ssl:external_openssl" ]
+  } else {
+    deps = [
+      "//third_party/boringssl",
+    ]
+  }
 }
-- 
2.4.4

