From b17939a2d0c21ae5cfd6956ed98b47789d4dee8f Mon Sep 17 00:00:00 2001
From: Qiu Jainlin <jianlin.qiu@intel.com>
Date: Mon, 4 Jan 2021 15:49:43 +0800
Subject: [PATCH] Use OpenSSL for usrsctp

---
 usrsctp/BUILD.gn | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/usrsctp/BUILD.gn b/usrsctp/BUILD.gn
index 0c5947f1785..7aff0d50687 100644
--- a/usrsctp/BUILD.gn
+++ b/usrsctp/BUILD.gn
@@ -3,6 +3,7 @@
 # found in the LICENSE file.
 
 import("//build/toolchain/toolchain.gni")
+import("//build_overrides/ssl/ssl.gni")
 import("//testing/libfuzzer/fuzzer_test.gni")
 
 config("usrsctp_public_config") {
@@ -161,7 +162,14 @@ static_library("usrsctp") {
   if (use_fuzzing_engine) {
     defines += [ "INVARIANTS" ]
   }
-  deps = [ "//third_party/boringssl" ]
+
+  if (owt_use_openssl) {
+    configs += [ "//build_overrides/ssl:external_openssl" ]
+  } else {
+    deps = [
+      "//third_party/boringssl",
+    ]
+  }
 }
 
 fuzzer_test("usrsctp_fuzzer_listen") {
-- 
2.28.0.windows.1

