From 06152513e72cf0dc998af85d67185861b93ecfe2 Mon Sep 17 00:00:00 2001
From: zhangyihui1 <yihuizhx@intel.com>
Date: Fri, 19 Jun 2020 18:57:39 +0800
Subject: [PATCH] Adjusted jni_generator.py to fit OWT code structure.

---
 android/jni_generator/jni_generator.py | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/android/jni_generator/jni_generator.py b/android/jni_generator/jni_generator.py
index 5a93cac..2a60486 100755
--- a/android/jni_generator/jni_generator.py
+++ b/android/jni_generator/jni_generator.py
@@ -1308,6 +1308,10 @@ $TEST_METHODS
     if not self.options.includes:
       return ''
     includes = self.options.includes.split(',')
+    for i in range(len(includes)):
+      if includes[i] == '../../../../../../../../sdk/android/src/jni/jni_generator_helper.h':
+        includes[i] = '../../../../../../../../third_party/webrtc/sdk/android/src/jni/jni_generator_helper.h'
+
     return '\n'.join('#include "%s"' % x for x in includes) + '\n'
 
   def GetOpenNamespaceString(self):
@@ -1673,6 +1677,12 @@ def GetScriptName():
       break
   return os.sep.join(script_components[base_index:])
 
+def _MoveJNIHeader4OWT(output_dir, output_file):
+  # Copy jni files.
+  new_path = output_dir.replace('/third_party/webrtc', '')
+  if not os.path.exists(new_path):
+    os.makedirs(new_path)
+  shutil.copy2(output_file, new_path)
 
 def _RemoveExistingHeaders(path):
   if os.path.exists(path) and os.path.isdir(path):
@@ -1785,6 +1795,7 @@ See SampleForTests.java for more details.
 
     for java_path, header_path in zip(input_files, output_files):
       GenerateJNIHeader(java_path, header_path, args)
+      _MoveJNIHeader4OWT(header_path, java_path)
   finally:
     shutil.rmtree(temp_dir)
 
-- 
2.7.4

