From 9c3dae0473f6d2f1011a0917436a536af7deec0c Mon Sep 17 00:00:00 2001
From: Jianjun Zhu <jianjun.zhu@intel.com>
Date: Sun, 14 Feb 2021 10:37:08 +0800
Subject: [PATCH] Fix examples path error.

---
 config/linux/gtk/BUILD.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/linux/gtk/BUILD.gn b/config/linux/gtk/BUILD.gn
index 5491aef73..6bae2dcf0 100644
--- a/config/linux/gtk/BUILD.gn
+++ b/config/linux/gtk/BUILD.gn
@@ -38,7 +38,7 @@ group("gtk") {
     "//remoting/host/it2me:remote_assistance_host",
     "//remoting/host/linux",
     "//remoting/test:it2me_standalone_host_main",
-    "//webrtc/examples:peerconnection_client",
+    "//third_party/webrtc/examples:peerconnection_client",
   ]
 
   public_configs = [ ":gtk_internal_config" ]
-- 
2.30.0

