From e0357ec7dfded3d287d71eccee1d20c6ceca0cfd Mon Sep 17 00:00:00 2001
From: Qiu Jainlin <jianlin.qiu@intel.com>
Date: Thu, 7 May 2020 22:14:11 +0800
Subject: [PATCH] Remove unused gni for av1 build

---
 libaom/options.gni | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libaom/options.gni b/libaom/options.gni
index a184f1eb9fd..56258965a50 100644
--- a/libaom/options.gni
+++ b/libaom/options.gni
@@ -1,4 +1,4 @@
-import("//build/config/gclient_args.gni")
+#import("//build/config/gclient_args.gni")
 
 declare_args() {
   # Enable encoding and decoding AV1 video files.
-- 
2.24.1.windows.2

